docs/quickstart/services.rst

```diff
@@ -29,7 +27,7 @@ Forward the port used by Grafana:
 
 .. code-block:: shell
 
-   user@your-host $ kubectl --namespace metalk8s-monitoring port-forward svc/grafana 3000
+   user@your-host $ kubectl port-forward -n metalk8s-monitoring prometheus-operator-grafana-54b8dc5844-sf5x4 3000
 
 Then open your web browser and navigate to ``http://localhost:3000``
```

Comment:
If you mention a `<local_port>` as suggested above, this would become:

```suggestion
Then open your web browser and navigate to ``http://localhost:<local_port>``
```